UPGRADE.md

```diff
@@ -1572,13 +1572,9 @@ Here's how to migrate the *Altering the Form Values before Submitting* example f
 import * as React from 'react';
 import { useCallback } from 'react';
 import { useForm } from 'react-final-form';
-import { SaveButton, Toolbar, useCreate, useRedirect } from 'react-admin';
+import { SaveButton, Toolbar } from 'react-admin';
```

Comment:
Wouldn't it be better for version 3 upgrade guides to be in a separate file ?

Reply:
I think we'll remove the instructions for upgrading to version 3, they are obsolete